File: javascript/1047.js
/*Tempo de Jogo com Minuto*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split(" ");

let Hora = Number(lines.shift());
let Minuto = Number(lines.shift());
let HoraFinal = Number(lines.shift());
let MinutoFinal = Number(lines.shift());

let Calculo = (HoraFinal - Hora);
let Calculation = (MinutoFinal - Minuto);

if(Calculation < 0){
  Calculation += 60;
  Calculo -= 1;
}
if(Calculo < 0){
  Calculo += 24;
}
if(Calculation == 0 && Calculo == 0){
  console.log("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)");
}else{
  console.log("O JOGO DUROU " + Calculo + " HORA(S) E " + Calculation + " MINUTO(S)");
}

File: python/condicoes/1047.py
#1047 TEMPO DE JOGO COM MINUTOS
valores = input().split(" ")
HoraInicial = int(valores[0])
MinutoInicial = int(valores[1])
HoraFinal = int(valores[2])
MinutoFinal = int(valores[3])

calculoH = HoraFinal-HoraInicial  
calculoM = MinutoFinal-MinutoInicial

if calculoM < 0:
    calculoM += 60
    calculoH -= 1
if calculoH < 0:
    calculoH += 24

if HoraInicial == HoraFinal and MinutoInicial == MinutoFinal:
    print("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
else:
    print(f"O JOGO DUROU {calculoH} HORA(S) E {calculoM} MINUTO(S)")